services/login_service/app/requirements.txt

```diff
@@ -0,0 +1,35 @@
+blinker==1.8.2
+cachetools==5.5.0
+certifi==2024.8.30
+charset-normalizer==3.3.2
+click==8.1.7
+Flask==3.0.3
```

Comment:
I think flask is not needed anymore. Also it's best not to do `pip freeze` to create this file.
It's best to have here only dependencies you are installing directly. For bigger project this can result in conflicts